packages/components/psammead-brand/README.md

```diff
@@ -1,25 +1,41 @@
 # Brand &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-brand.svg)](https://www.npmjs.com/package/@bbc/psammead-brand) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
 
-## Description	
+## Description
+
 The `Brand` component provides the BBC News logo (as SVG), nested inside a styled span, link and div. The link is currently hardcoded to "https://www.bbc.co.uk/news". `Brand` takes a `brandName` as a prop. This prop is passed to a [VisuallyHiddenText](../VisuallyHiddenText) component, nested inside Brand. Note that this does not currently affect the branding itself, which always renders as `BBC NEWS`.
 
-## Usage 
-```
+## Usage
+
+| Prop    | Description     |
+| ------- | --------------- |
+| `brandName`  | Text equivalent of the Brand name, e.g. `BBC News`. Should be literal translation of the SVG text. Used for visually hidden text for users of Assistive Technologies. |
+| `children`   | React elements for Brand SVG paths, with all relevant SVG attributes as camel-cased (e.g. `stroke-width` as `strokeWidth`, [see React Docs](https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes)) |
+| `svgWidth` | A number representing the width of the SVG, given that the height is fixed to 24px. Used for the viewbox width and width attribute of the SVG.  |
```

Comment:
I question the usefulness of the component if you have to pass an SVG to it 🤔 for me, I'd simplify the API to be:

```
name - as per brandName, but refactored name as we don't need to repeat "brand"
children - optional
```

If we recognise the `name` passed, we render the corresponding SVG (which is stored somewhere inside the component), which has its height & width defined too. So simple use-case would be:

```jsx
<Brand name="BBC News" />
```

For unrecognised brands, I'd propose this API:

```jsx
<Brand name="BBC Foo">
    <svg ...bla>
    </svg>
</Brand>
```

^ and in that case, we explicitly put in the documentation that any custom SVG has the width/height we require.

Reply:
My main reason for not including the SVG path in this brand component was that we wouldn't want to bloat this component with 40+ service SVGs.
Would we want to add them into this component?